albumentations/augmentations/functional.py

```diff
@@ -1256,3 +1256,20 @@ def py3round(number):
 
 def noop(input_obj, **params):
     return input_obj
+
+
+def swap_tiles_on_image(image, tiles):
+    """
+    Swap tiles on image.
+
+    Args:
+        image (np.ndarray): Input image.
+        tiles (np.ndarray): array of tuples with bounding boxes
```

Comment:
indexing is not obvious here, please make a class or namedtuple or something like this because it's not clear what is tiles without digging into a code

Reply:
I supplemented the description